### PR TITLE
Contrast building color with background color

### PR DESCRIPTION
**Pull request by lemieux**
It's pretty much a must fix for existing user since they'll have brandColor = #fff and it would end up white on white for the business icon. This now uses the same logic as other parts of the UI to contrast the content of the icon.

@mspensieri @dannytranlx @Mario54 @jugarrit @alavers @chloepouprom

### CODE REVIEW

**Comment by jugarrit** [comment]
👍

File: src/js/components/default-app-icon.jsx [hunk 1]
```diff
@@ -1,68 +1,73 @@
...
+import React, { Component, PropTypes } from 'react';
+import { SK_DARK_CONSTRAST } from '../constants/styles';
 
 export class DefaultAppIcon extends Component {
-    static defaultProps = {
-        style: {
-            fill: SK_PURPLE
-        }
+    static propTypes = {
+        color: PropTypes.string
```
Comment (chloepouprom) [inline src/js/components/default-app-icon.jsx]: Do we still need this? Since the color is in the context

**Comment by Mario54** [comment]
👍

File: src/js/components/default-app-icon.jsx (hunk 1 above)
Comment (lemieux) [inline src/js/components/default-app-icon.jsx]: eh good catch, it should go away

**Pull request merged**